### DISCUSSION TITLE
Interface enchancements

### DISCUSSION THREAD

**Discussion by TuTAH1** [BlueBird67/BlackScreens#9] [Ideas]
My suggestions are:

- [ ] Remove "Exit" and "Minimize" button, becouse the window already have buttons for this actions on the top (tautology)
- [ ] Border "Set image" button with grayish beckground, becouse currently it don't look like a button

**Comment by BlueBird67** [comment]
Hello, thank you for this interesting remarks and propositions. I will look at it whenever I will have some time beside work.
